File: c/Colleen/Colleen.c
#include <stdio.h>

/*
	comment outside
*/

void print_colleen(char *s)
{
	printf(s, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 34, s, 34, 10, 10, 10, 10, 10, 10, 10, 10);
}

int main(void)
{
	char s[] = "#include <stdio.h>%c%c/*%c	comment outside%c*/%c%cvoid print_colleen(char *s)%c{%c	printf(s, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 10, 34, s, 34, 10, 10, 10, 10, 10, 10, 10, 10);%c}%c%cint main(void)%c{%c	char s[] = %c%s%c;%c%c	/*%c		comment inside main%c	*/%c	print_colleen(s);%c	return 0;%c}%c";

	/*
		comment inside main
	*/
	print_colleen(s);
	return 0;
}

File: python/Colleen/Colleen.py
#!/usr/bin/python

#
#	comment outside
#

def print_colleen(s):
	print(s % (10, 10, 10, 10, 10, 10, 10, 37, 10, 10, 10, 34, s, 34, 10, 10, 10, 10, 10, 10, 10))

def main():
	s = "#!/usr/bin/python%c%c#%c#	comment outside%c#%c%cdef print_colleen(s):%c	print(s %c (10, 10, 10, 10, 10, 10, 10, 37, 10, 10, 10, 34, s, 34, 10, 10, 10, 10, 10, 10, 10))%c%cdef main():%c	s = %c%s%c%c%c#%c#	comment inside main%c#%c	print_colleen(s)%c%cmain()"

#
#	comment inside main
#
	print_colleen(s)

main()
